docs/react/tutorial/08-create-libs.md

```diff
@@ -87,21 +86,20 @@ myorg/
 │   └── ui/
 │       ├── src/
 │       │   ├── lib/
-│       │   │   ├── ui/
-│       │   │   │   ├── ui.css
-│       │   │   │   ├── ui.spec.tsx
-│       │   │   │   └── ui.tsx
 │       │   │   └── todos/
-│       │   │       ├── todos.css
-│       │   │       ├── todos.spec.tsx
-│       │   │       └── todos.tsx
+│       │   │   │   ├── todos.css
+│       │   │   │   ├── todos.spec.tsx
+│       │   │   │   └── todos.tsx
+│       │   │   ├── ui.css
+│       │   │   ├── ui.spec.tsx
+│       │   │   └── ui.tsx
 │       │   └── index.ts
 │       ├── jest.conf.js
-│       ├── tsconfig.app.json
+│       ├── project.json
 │       ├── tsconfig.json
+│       ├── tsconfig.lib.json
 │       └── tsconfig.spec.json
 ├── tools/
-├── workspace.json
```

Comment:
Why was this removed?

Reply:
It was not there in the previous treeview on the same page. So I decided to keep it consistent here. But I will add it to both places now.